tox.ini

```diff
@@ -70,6 +70,7 @@ deps =
     six
     future
     pyyaml
+    protobuf == 3.19.6
```

Comment:
This is currently required, as otherwise, SageMaker imports fail with error:
```
    import sagemaker.amazon.common
.tox/py37/lib/python3.7/site-packages/sagemaker/amazon/common.py:23: in <module>
  from sagemaker.amazon.record_pb2 import Record
.tox/py37/lib/python3.7/site-packages/sagemaker/amazon/record_pb2.py:52: in <module>
    file=DESCRIPTOR,
.tox/py37/lib/python3.7/site-packages/google/protobuf/descriptor.py:560: in __new__
    _message.Message._CheckCalledFromGeneratedFile()
E   TypeError: Descriptors cannot not be created directly.
E   If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
E   If you cannot immediately regenerate your protos, some other possible workarounds are:
E    1. Downgrade the protobuf package to 3.20.x or lower.
E    2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).
E
E   More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates
```

Upgrading SageMaker version did not resolve the issue, so currently we need to pin the version, and consider a complete upgrade of dependencies in a separate PR.